nucliadb_writer/nucliadb_writer/resource/field.py

```diff
@@ -230,8 +215,24 @@ async def parse_file_field(
 
     processing = get_processing()
 
-    # Send to process
-    toprocess.filefield[key] = await processing.convert_filefield_to_str(file_field)
+    if skip_store:
```

Comment:
I don;t understand, skiping store means not storing on nucliadb

Reply:
sorry, I shouldn't have mixed a new feature + a refactor in the same PR. I will close and reopen after your refactor